starter-code/server.js

```diff
@@ -92,8 +92,8 @@ app.put('/articles/:id', function(request, response) {
     client.query(
       `UPDATE articles
       SET title = $2, category = $3, "publishedOn" = $4, body = $5
-      WHERE author_id = $1;`,
-      [request.body.author_id, request.body.title, request.body.category, request.body.publishedOn, request.body.body]
+      WHERE article_id = $1;`,
+      [request.params.id, request.body.title, request.body.category, request.body.publishedOn, request.body.body]
```

Comment:
Yep! And here you still want to update `author_id` because it's possible that `author_id` is updated in `authors` table, but then the `author_id` in article table isn't getting updated here.

Hence, you'll have `SET author_id=$x`